package.json

```
@@ -39,9 +39,9 @@
 		"@types/node": "^16.0.0"
 	},
 	"engines": {
-		"pnpm": ">=6.0.0",
+		"pnpm": ">=7.0.0",
```

We might even consider bumping this to contain a minimum minor version, just in case bug fixes/additions were made which we utilize and don't realize are missing in older minor versions (unlikely but possible).

```suggestion
		"pnpm": ">=7.14.0",
```